src/pages/Improbability/Improbability.component.tsx

```diff
@@ -0,0 +1,69 @@
+import Typography from '@mui/material/Typography';
+import { useForm } from 'react-hook-form';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { DataTable } from '../../components/DataTable/DataTable.component';
+import { PageView } from '../../components/PageView/PageView.component';
+import {
+  ImprobabilityComponentProps,
+  ImprobabilityFormValues,
+  ImprobabilityRowData,
+} from './Improbability.types';
+import { ControlledDropdown } from '../../components/Dropdown/Dropdown.controlled';
+import {
+  coinsListColumns,
+  improbabilityDropdownOptions,
+} from './Improbability.utils';
+
+export const ImprobabilityComponent = ({
+  state,
+  coins,
+}: ImprobabilityComponentProps) => {
+  const { setValue, control } = useForm<ImprobabilityFormValues>({
+    mode: 'onChange',
+    defaultValues: { network: '' },
+  });
+
+  return (
+    <PageView
+      title={
+        <Typography variant="h2" padding={1}>
+          Improbability Drive
+        </Typography>
+      }
+    >
+      <Box
+        sx={{
+          width: '100%',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          gap: 2,
+        }}
+      >
+        <ControlledDropdown
+          name="network"
+          placeholder="Choose Network"
+          control={control}
+          options={improbabilityDropdownOptions}
+          setValue={setValue}
+        />
+        <Button
+          sx={{ height: '3rem', width: '12rem', ml: 'auto' }}
```

Comment:
For consistency, I suggest using mui spacings for margins and paddings and px for widths and heights(so always just numbers withour quotes).